cpp/src/integrations/cudf/utils.cpp

```diff
@@ -37,6 +45,43 @@ struct str_cudf_column_scalar_fn {
         RAPIDSMPF_FAIL("not implemented");
     }
 };
+
+struct cudf_column_data_size_fn {
+    template <typename T>
+        requires(cudf::is_fixed_width<T>())
+    size_t operator()(cudf::column_view const& col, rmm::cuda_stream_view) {
+        return static_cast<size_t>(col.size()) * cudf::size_of(col.type())
+               + bitmask_size(col);
+    }
+
+    // string type specialization
+    template <typename T>
+        requires(std::is_same_v<T, cudf::string_view>)
+    size_t operator()(cudf::column_view const& col, rmm::cuda_stream_view stream) {
+        cudf::strings_column_view sv(col);
+        return static_cast<size_t>(sv.chars_size(stream)) + bitmask_size(col);
```

Comment:
Note that this launches a blocking kernel on `stream`...

Reply:
I felt like its inevitable, because we dont have access to the rmm buffer sizes with the column view. Isnt it?